docs(spring-boot): Adding the spring boot experiments docs (#3834)
litmuschaos/litmus: mkdocs/docs/experiments/categories/spring-boot/spring-boot-cpu-stress.md

            verbs: ["create","list","get","patch","update","delete"]
        ---
        apiVersion: rbac.authorization.k8s.io/v1
        kind: RoleBinding
        metadata:
          name: spring-boot-cpu-stress-sa
          namespace: default
          labels:
            name: spring-boot-cpu-stress-sa
            app.kubernetes.io/part-of: litmus
        roleRef:
          apiGroup: rbac.authorization.k8s.io
          kind: Role
          name: spring-boot-cpu-stress-sa
        subjects:
          - kind: ServiceAccount
            name: spring-boot-cpu-stress-sa
            namespace: default
        ```
        Use this sample RBAC manifest to create a chaosServiceAccount in the desired (app) namespace. This example consists of the minimum necessary role permissions to execute the experiment.

## Experiment tunables

??? info "check the experiment tunables"

    <h2>Mandatory Fields</h2>

    <table>
      <tr>
        <th> Variables </th>
        <th> Description </th>
        <th> Notes </th>
      </tr>
      <tr>
        <td> CM_PORT </td>
        <td> It contains port of the spring boot application </td>
        <td> </td>
      <tr>
        <td> CPU_LOAD_FRACTION </td>

>         <td> It contains fraction of cpu to be stressed, 0.95 equals 95%</td>

Reviewer: ```suggestion
        <td> It contains fraction of CPU to be stressed, Eg: 0.95 equals 95%</td>
```

>         <td> default value is 0.9. It supports value in range [0.1,1.0]</td>

Reviewer: ```suggestion
        <td> Default value is 0.9. It supports a value in range [0.1,1.0]</td>
```

      </tr>
      </tr>
    </table>
    
    <h2>Optional Fields</h2>

    <table>
      <tr>
        <th> Variables </th>
        <th> Description </th>
        <th> Notes </th>
      </tr>
      <tr>
        <td> CM_LEVEL </td>
        <td> It contains number of requests are to be attacked, n value means nth request will be affected </td>

>         <td> Defaults value: 1, it lies in [1,10000] range </td>

Reviewer: ```suggestion
        <td> Default value: 1, it lies in [1,10000] range </td>
```

      </tr>
      <tr>
        <td> CM_WATCHED_CUSTOM_SERVICES </td>
        <td> It limits watched packages/classes/methods, it contains comma seperated list of fully qualified packages(class and/or method names)</td>

>         <td> ByDefault it is empty list, which means it target all services </td>

Reviewer: ```suggestion
        <td> Default is an empty list, which means it will target all services </td>
```

      </tr>
      <tr>
        <td> CM_WATCHERS </td>
        <td> It contains comma separated list of watchers from the following watchers list [controller, restController, service, repository, component, webClient] </td>

>         <td> ByDefault it is <code> restController </code></td>

Reviewer: ```suggestion
        <td> Default is <code> restController </code></td>
```
At other places
